.babelrc

```diff
@@ -11,6 +11,13 @@
         "preprocess": false,
       }
     ],
+    ["module-resolver", {
+      "root": ["./"],
+      "alias": {
+        "test": "./test",
+        "underscore": "lodash"
```

Comment:
Why is loadash defined as a path here?

Reply:
I just copied the settings from the packet site which they said I should put this in babelrc